environments/india/public.yml

```diff
@@ -117,6 +117,9 @@ localsettings:
   EMAIL_SMTP_PORT: 587
   EMAIL_USE_TLS: yes
   ENABLE_DRACONIAN_SECURITY_FEATURES: yes
+  ADD_CAPTCHA_FIELD_TO_FORMS: True
+  DISABLE_AUTOCOMPLETE_ON_SENSITIVE_FORMS: True
+  CUSTOM_PASSWORD_STRENGTH_MESSAGE: 'Password Requirements: 1 special character, 1 number, 1 capital letter, minimum length of 8 characters.'
```

Comment:
Why are we adding these to `india`? I don't think they should be present there.

Reply:
that is a good point Simon. And a parallel conversation is needed, to check on why we even have the draconian features available on India, I guess that configuration was not removed after moving icds-cas out of india.
I will be taking that up. These changes are currently made in the PR to just wait for that confirmation.

Reply:
Correct, they were needed prior to having a dedicated icds-staging env. They can (and should) be removed.

Reply:
Hey Simon, 
Updated. Please share review on recent changes.